validator/pom.xml

```diff
@@ -65,4 +65,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
Do we need this? It doesn't seem related.

Reply:
Indeed, it's not related, it is to fix `ValidateConfigTest` on non english system. Do you want me to remove it?